launch/sick_tim_tcp.launch

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<launch>
+   
+    <!--Starts a robot state publisher.-->
+    <!--
+  <param name="robot_description" command="$(find xacro)/xacro.py '$(find sick_tim)/urdf/example.urdf.xacro'" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+    -->
+
+  <node name="sick_tim551_2050001" pkg="sick_tim" type="sick_tim551_2050001" respawn="false" output="screen">
+    <!-- default values: -->
+
+      <param name="min_ang" type="double" value="-2.35619449019" />
+      <param name="max_ang" type="double" value="2.35619449019" />
+      <param name="intensity" type="bool" value="True" />
+      <param name="skip" type="int" value="0" />
+      <param name="frame_id" type="str" value="laser" />
+      <param name="time_offset" type="double" value="-0.001" />
+      <param name="publish_datagram" type="bool" value="False" />
+      <param name="subscribe_datagram" type="bool" value="false" />
+
+    <!-- Enables TCP instead of USB connection; `hostname` is the host name or IP address of the laser scanner-->
+    <!-- In cases where a race condition exists and the computer boots up before the TIM is ready, increase 'timelimit.' -->
+      <param name="hostname" type="string" value="192.168.1.14" />
+      <param name="port" type="string" value="2112" />
+      <param name="timelimit" type="string" value="5" />
+
+  </node>
+</launch>
```

Comment:
I think we don't need the launch file; we can add the timelimit param to sick_tim551_2050001.launch instead (commented out, like the rest).